src/services/authentication/logic/whitelist.js

```diff
@@ -70,19 +70,22 @@ const redisClientExists = () => !!getRedisClient();
  */
 const isTokenAvailable = (token) => !!token;
 
+// eslint-disable-next-line consistent-return
 const addTokenToWhitelist = async (redisIdentifier, privateDevice = false) => {
-	const redisData = getRedisData({ privateDevice });
-	const { expirationInSeconds } = redisData;
-	await redisSetAsync(redisIdentifier, JSON.stringify(redisData), 'EX', expirationInSeconds);
-	return { ttl: expirationInSeconds };
+	if (redisClientExists()) {
```

Comment:
and if not?

Reply:
This is done analogous to the other functions. If Redis is not available, the code is skipped and nothing is returned.

Reply:
ok